### PR TITLE
Logging for art-bot.py, update logging config

### PR DESCRIPTION
**Pull request by ashwindasr**
- Add logs for art-bot.py
- Update logging config as suggested by @locriandev

### CODE REVIEW

File: art-bot.py [hunk 1]
```diff
@@ -92,8 +91,7 @@ def respond(client, event):
         data = event
         web_client = client
 
-        print('\n----------------- DATA -----------------\n')
-        pprint.pprint(data)
```
Comment (locriandev) [inline art-bot.py:96]: it might be that in this case we want to keep the `pprint` to have a better looking log

File: art-bot.py [hunk 2]
```diff
@@ -125,8 +123,8 @@ def respond(client, event):
 
         plain_text = extract_plain_text({"data": data}, alt_username)
 
-        print(f'Gating {from_channel}')
```
Comment (locriandev) [inline art-bot.py:128]: Maybe these could be logged at `DEBUG` level

Comment (ashwindasr) [reply art-bot.py:128]: Done!

File: art-bot.py (hunk 1 above)
Comment (ashwindasr) [reply art-bot.py:96]: Fixed! The `ppring.pformat` function helped.

**Pull request merged**